book/page_creation.rst

```diff
@@ -434,30 +434,41 @@ worked inside the two most important directories:
 else). As you get more advanced, you'll learn what can be done inside each
 of these.
 
-The ``app/`` directory also holds a few other things, like the cache directory
-``app/cache/``, the logs directory ``app/logs/`` and ``app/AppKernel.php``,
-which you'll use to enable new bundles (and one of a *very* short list of
+The ``app/`` directory also holds some other things, like ``app/AppKernel.php``,
+which you'll use to enable new bundles (this is one of a *very* short list of
 PHP files in ``app/``).
 
 The ``src/`` directory has just one directory - ``src/AppBundle`` -
 and everything lives inside of it. A bundle is like a "plugin" and you can
 `find open source bundles`_ and install them into your project. But even
-*your* code lives in a bundle - typically ``AppBundle`` (though there's
-nothing special about ``AppBundle``). To find out more about bundles and
+*your* code lives in a bundle - typically *AppBundle* (though there's
+nothing special about AppBundle). To find out more about bundles and
 why you might create multiple bundles (hint: sharing code between projects),
 see the :doc:`Bundles </book/bundles>` chapter.
 
 So what about the other directories in the project?
 
-``vendor/``
-    Vendor (i.e. third-party) libraries and bundles are downloaded here by
-    the `Composer`_ package manager.
-
 ``web/``
     This is the document root for the project and contains any publicly accessible
     files, like CSS, images and the Symfony front controllers that execute
     the app (``app_dev.php`` and ``app.php``).
 
+``tests/``
+    The automatic tests (e.g. Unit tests) of your application live here.
+
+``bin/``
+    The "binary" files live here. The most important one is the ``console``
+    file which is used to execute Symfony commands via the console.
+
+``var/``
+    This is were automatically created files are stored, like cache files
+    (``var/cache/``) and logs (``var/logs/``).
+
+``vendor/``
+    Vendor (i.e. third-party) libraries and bundles are downloaded here by
```

Comment:
Instead of:

```
Vendor (i.e. third-party) libraries and bundles are downloaded ...
```

What about:

```
Third-party libraries, packages and bundles needed by the project are downloaded ...
```